src/main/scala/com/gu/emailservices/EmailFields.scala

```diff
@@ -1,35 +1,41 @@
 package com.gu.emailservices
 
-import io.circe.{Json, Printer}
 import org.joda.time.DateTime
 import org.joda.time.format.DateTimeFormat
+import io.circe.generic.auto._
+import io.circe.syntax._
+import com.gu.salesforce.Salesforce.SfContactId
+
+case class EmailPayloadContactAttributes(SubscriberAttributes: Map[String, String])
```

Comment:
the comments on this line seem to have been collapsed for some reason https://github.com/guardian/support-workers/pull/139#pullrequestreview-156844346

Reply:
I've replied 😄